### PR TITLE
[opt](config) rename alter_inverted_index_worker_count to alter_index_worker_count, and add config alter_index_worker_count usage to the doc

### PR DESCRIPTION
**Pull request by Tanya-W**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Tanya-W** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 51.24 seconds
 stream load tsv:          507 seconds loaded 74807831229 Bytes, about 140 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          28.9 seconds inserted 10000000 Rows, about 346K ops/s
 storage size: 17169081220 Bytes

**Review by xiaokang — DISMISSED** [review]
LGTM

File: docs/zh-CN/docs/advanced/alter-table/schema-change.md [hunk 1]
```diff
@@ -289,6 +289,8 @@ SHOW ALTER TABLE COLUMN\G;
 
 - `alter_tablet_worker_count`：在 BE 端用于执行历史数据转换的线程数。默认为 3。如果希望加快 Schema Change 作业的速度，可以适当调大这个参数后重启 BE。但过多的转换线程可能会导致 IO 压力增加，影响其他操作。该线程和 Rollup 作业共用。
 
+- `alter_index_worker_count`：在 BE 端用于执行历史数据构建索引的线程数。默认为 3。如果希望加快 Index Change 作业的速度，可以适当调大这个参数后重启 BE。但过多的线程可能会导致 IO 压力增加，影响其他操作。
```
Comment (xiaokang) [inline docs/zh-CN/docs/advanced/alter-table/schema-change.md:292]: 说明一下目前只支持倒排索引

Comment (Tanya-W) [reply docs/zh-CN/docs/advanced/alter-table/schema-change.md]: updated

**Comment by Tanya-W** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 50.63 seconds
 stream load tsv:          507 seconds loaded 74807831229 Bytes, about 140 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          28.9 seconds inserted 10000000 Rows, about 346K ops/s
 storage size: 17170982605 Bytes

**Review by qidaye — APPROVED** [review]
LGTM

**Review by xiaokang — APPROVED** [review]
LGTM

**Pull request merged**